KIPs/kip-237.md

```diff
@@ -0,0 +1,55 @@
+---
+kip: 237
+title: Storage Compression
+author: Lake (@hyunsooda), Ollie (@blukat29)
+discussions-to: TBD
+status: Draft
+category: Storage
+created: 2024-12-26
+---
+
+## Abstract
+Reduce disk storage by compressing data, cutting its size by 50% over.
```

Comment:
Let's make it more informative.

```
Compress the block header, body, and receipts data in its persistent form to reduce the disk footprint. In case of a state-migrated full mainnet node, its database size can be reduced by NN% from N.NTB to N.NTB. With an efficient data structure, those data could be compressed without loss at a minimal query overhead.
```

Reply:
Done https://github.com/kaiachain/kips/pull/37/commits/ffd2fcf967ec51a2b8c5a25f4d376f69ccd4b343